docs/veritas-10.3/NetBackup Troubleshooting API.html

```
@@ -0,0 +1,88 @@
+        margin:0;
+        background: #fafafa;
+      }
+    </style>
+  </head>
+
+  <body>
+    <div style="border-bottom: 2px solid rgba(0,0,0,.15); background-color:#ffffff">
+        <div class="swagger-header-div">
+            <table style='width: 98%'>
+                <tr>
+                <td><img class="header-img" src="/public/images/swagger_icon.png"></img><p class="header-title"><strong>NetBackup</strong></p></td>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </img> ]

(tag-pair)